website/docs/admin_colorspace.md

```diff
@@ -0,0 +1,193 @@
+
+### 1. Example of usage - [DCCs](admin_colorspace#used-acronyms) are set to native linear rec709 colorspace
+
+#### Actual configuration:
+A studio is having on-going production with project configured imageio at op 3.15. All hosts are set to `linear rec709` space with following combination of settings:
+- Nuke host [**CMS**](admin_colorspace#used-acronyms) custom config was not activated and it was used as `nuke-default`
+- Maya was set to internal ocio config in v2 `maya-scene-linear-rec709`
+- Global imageio custom config was set to `nuke-default`
+
+#### Required action after the upgrade:
+No need to do anything. Be aware that once the Global [**CMS**](admin_colorspace#used-acronyms) is enabled, the default config is set to `aces_1.2` and it will be applied to all hosts. If you want to keep the same behavior as before the upgrade, you need to set the Global [**CMS**](admin_colorspace#used-acronyms) to `nuke-default` config or disable it.
+
+### 2. Example of usage - [DCCs](admin_colorspace#used-acronyms) are set to native ACES OCIO config
+#### Actual configuration:
+A studio is having on-going production with project configured imageio at op 3.15. All hosts are set to `ACES 1.2` space with following combination of settings:
+   - Nuke host [**CMS**](admin_colorspace#used-acronyms) custom config was not activated and it was used OCIO with provided `aces_1.2`
+   - Maya was set to internal ocio config in v2 `maya-aces_1.2`
+   - Global imageio custom config was set to distributed `aces_1.2`
+
+#### Required action after the upgrade:
+No need to do anything. Be aware that once the Global [**CMS**](admin_colorspace#used-acronyms) is enabled, the default config is set to `aces_1.2` and it will be applied to all hosts. If you want to keep the same behavior as before the upgrade, you need to set the Global [**CMS**](admin_colorspace#used-acronyms) to `aces_1.2` config or disable it.
+
+### 3. Example of usage - Global custom ocio config used
+#### Actual configuration:
+A studio is having on-going production with project configured imageio at op 3.15. All hosts are set to custom config path via OCIO env var with following combination of settings:
+   - Nuke host [**CMS**](admin_colorspace#used-acronyms) is having defined creator nodes to corresponding colorspace name found in provided OCIO config
+   - Maya is having activated OCIO v2 via OCIO env var and it is using provided OCIO config
+   - Global File rules were defined to:
+        - capture all render family subsets to be **acescg** colorspace
+        - capture all plates family subsets to be **acescc** colorspace
+        - capture all mp4 extensions to be **Output Rec.709** colorspace
+
+#### Required action after the upgrade:
+1. Enable Global [**CMS**](admin_colorspace#used-acronyms).
+2. Add Studio OCIO config into any folder within Root defined storage.
```

Comment:
> into any folder within Root defined storage.

This technically is NOT a requirement. It just needs to be accessible by all machines. It could also contain a global environment variable, right? Or in a windows only single physical location studio it could technically be even a single network path.

This is much better described as "Set the global OCIO config" and potentially mention to 'as per the recommended method' and then linking to that section of what to keep in mind.

Reply:
Yes not required and the new text will mention it.